dev/integration_tests/flutter_gallery/lib/demo/material/menu_demo.dart

```diff
@@ -181,6 +177,10 @@ class MenuDemoState extends State<MenuDemo> {
                   child: Text(_simpleValue3),
                 ),
               ],
+              child: ListTile(
```

Comment:
This one isn't an improvement. the lint needs to be smarter about realizing that a function returning Widget objects is just as much a child generator as `child`.